apps/condo/domains/miniapp/schema/SendAppPushMessageService.js

```diff
@@ -0,0 +1,137 @@
+const { v4: uuid } = require('uuid')
+
+const conf = require('@open-condo/config')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT, FORBIDDEN } } = require('@open-condo/keystone/errors')
+const { getRedisClient } = require('@open-condo/keystone/redis')
+const { GQLCustomSchema, getByCondition } = require('@open-condo/keystone/schema')
+
+
+
+const { WRONG_FORMAT, NOT_FOUND, OPERATION_FORBIDDEN } = require('@condo/domains/common/constants/errors')
+const { LOCALES } = require('@condo/domains/common/constants/locale')
+const access = require('@condo/domains/miniapp/access/SendAppPushMessageService')
+const { B2CApp } = require('@condo/domains/miniapp/utils/serverSchema')
+const { MESSAGE_FORWARDED_TO_SUPPORT_TYPE, MESSAGE_TYPES, VOIP_INCOMING_CALL_MESSAGE_TYPE, CUSTOM_CONTENT_MESSAGE_PUSH_TYPE } = require('@condo/domains/notification/constants/constants')
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+
+const { USER_NOT_FOUND_ERROR, APP_NOT_FOUND_ERROR } = require('../constants')
+
+const DEFAULT_CACHE_TTL = 2
+
+const ERRORS = {
+    SEND_PUSH_TOO_OFTEN: {
+        mutation: 'sendAppPushMessage',
+        code: FORBIDDEN,
+        type: WRONG_FORMAT,
+        message: `${OPERATION_FORBIDDEN}] you already sent a push to this user less than ${DEFAULT_CACHE_TTL} seconds ago`,
+    },
+    USER_NOT_FOUND: {
+        mutation: 'sendAppPushMessage',
+        code: BAD_USER_INPUT,
+        type: USER_NOT_FOUND_ERROR,
+        message: 'Unable to find user by provided id.',
+    },
+    APP_NOT_FOUND: {
+        mutation: 'sendAppPushMessage',
+        code: BAD_USER_INPUT,
+        type: APP_NOT_FOUND_ERROR,
+        message: 'Unable to find app by provided id.',
+    },
+}
+
+const SendAppPushMessageService = new GQLCustomSchema('SendAppPushMessageService', {
+    types: [
+        {
+            access: true,
+            type: `enum SendAppPushMessageType { ${VOIP_INCOMING_CALL_MESSAGE_TYPE} }`,
+        },
+        {
+            access: true,
+            type: 'input SendAppPushMessageData { body: String!, title: String }',
+        },
+        {
+            access: true,
+            type: 'input SendAppPushMessageInput { dv: Int!, sender: SenderFieldInput!, app: B2CAppWhereUniqueInput!, user: UserWhereUniqueInput!, type: SendAppPushMessageType!, data: SendAppPushMessageData!, uniqKey: String }',
+        },
+        {
+            access: true,
+            type: 'type SendAppPushMessageOutput { id: String!, status: String! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canSendAppPushMessage,
+            schema: 'sendAppPushMessage(data: SendAppPushMessageInput!): SendAppPushMessageOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: argsData } = args
+                const { dv, sender, app, user, type, data, uniqKey } = argsData
```

Comment:
Use `const { checkDvAndSender } = require('@open-condo/keystone/plugins/dvAndSender')`
to check dv and sender